sdk/python/endpoints/batch/reading-multiple-inputs.ipynb

```diff
@@ -65,9 +65,9 @@
    },
    "outputs": [],
    "source": [
-    "subscription_id = \"18522758-626e-4d88-92ac-dc9c7a5c26d4\"\n",
-    "resource_group = \"Analytics.Aml.Experiments.Workspaces\"\n",
-    "workspace = \"aa-ml-aml-workspace\"\n",
```

Comment:
I must have checked in this notebook by mistake. This a notebook I used for testing different inputs. If we can remove it that would be great!